### ISSUE TITLE
[a11y]Consider increasing text contrast ratio in new_tab_button

### ISSUE THREAD

**Issue by priyanka0906** [mozilla-mobile/fenix#18780]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

### Actual behavior
![Screenshot_1617442916](https://user-images.githubusercontent.com/56197993/113474771-a1293a80-948f-11eb-9367-b7ef505fed52.png)

**Comment by priyanka0906** [comment]
@eliserichards  please take a look.
If it seems good to you , please assign this to me.

**Comment by eliserichards** [comment]
Thanks for filing this @priyanka0906! We will have our designers take a look before we start working on this 😄

**Comment by topotropic** [comment]
Thanks! The plus icon should be #fff so the contrast is 4.15:1 which passes AA 3:1 for large text. 

@priyanka0906  Could you verify that the plus is #fff? Thank you!

**Comment by priyanka0906** [comment]
@topotropic yes plus is #fff

**Comment by topotropic** [comment]
ok, that's great! Thanks, I'll close it then.
